src/server.lisp

```diff
@@ -0,0 +1,184 @@
+;;;; server.lisp
+;;;;
+;;;; Author: Eric Peterson
+;;;;
+;;;; Lisp mimic of the python JSON RPC Server.
+
+(in-package #:rpcq)
+
+
+(defun make-dispatch-table ()
+  (make-hash-table :test #'equal))
+
+(defun dispatch-table-add-handler (dispatch-table f
+                                   &key
+                                     (name (string f)))
+  "Add a function to a dispatch table, for use by an RPCQ server.  By default, a symbol passed in for F will be automatically converted into the name of F used in the dispatch table.  To manually specify a name (or to provide a name for a non-symbol value of F), use the keyword argument :NAME."
+  (setf (gethash (sanitize-name name) dispatch-table) f)
+  nil)
+
+
+(defun %pull-raw-request (receiver)
+  "Pulls a ZMQ request over the RECEIVER socket.  Returns a VALUES triple:
+
+* IDENTITY-FRAME: array of (INTEGER 256) that describes the intended recipient of a reply to this request.
```

Comment:
I suspect it's `(integer 0 (256))` == `(unsigned-byte 8)`. The type `(INTEGER 256)` is the set of integers >= 256.